AvatarImageView.podspec

```
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name              = "AvatarImageView"
-  s.version           = "2.1.2"
+  s.version           = "2.1.3"
```

```suggestion
  s.version           = "2.2"
```

This should be 2.2 as it contains breaking changes :)